pkg/kubectl/cmd/portforward.go

```diff
@@ -147,18 +148,19 @@ func (o *PortForwardOptions) Complete(f *cmdutil.Factory, cmd *cobra.Command, ar
 
 // Validate validates all the required options for port-forward cmd.
 func (o PortForwardOptions) Validate() error {
+	errors := []error{}
```

Comment:
`var errors []error` is generally the way we initialize in this scenario.  That avoids an allocation, but also more clearly indicates we want the default empty error to start with.